File: CloudBridge-WD/src/main/java/de/r3pt1s/CloudBridge/network/protocol/packet/LoginRequestPacket.java
// This class was created by r3pt1s
package de.r3pt1s.CloudBridge.network.protocol.packet;

public class LoginRequestPacket extends Packet {

    public Object server = "";

    @Override
    public void encode() {
        super.encode();
        put(server);
    }

    @Override
    public void decode() {
        super.decode();
        server = get();
    }

    @Override
    public Integer getId() {
        return ID_LOGIN_REQUEST;
    }

    public static LoginRequestPacket create(String server) {
        LoginRequestPacket pk = new LoginRequestPacket();
        pk.server = server;
        return pk;
    }
}

File: CloudSystem/src/Cloud/network/protocol/packet/LoginRequestPacket.php
<?php

namespace Cloud\network\protocol\packet;

class LoginRequestPacket extends Packet {

    public string $server = "";

    public function encode(): void {
        parent::encode();
        $this->put($this->server);
    }

    public function decode(): void {
        parent::decode();
        $this->server = $this->get();
    }

    public function getId(): int {
        return self::ID_LOGIN_REQUEST;
    }

    public static function create(string $server): self {
        $pk = new self;
        $pk->server = $server;
        return $pk;
    }
}